aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -0,0 +1,346 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBoxGroup,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  List,
+  RadioButtonGroup,
+  ResponsiveContext,
+  Select,
+  Text,
+  TextArea,
+  TextInput,
+} from 'grommet';
+import {
+  Checkmark,
+  FormClose,
+  FormNextLink,
+  FormPreviousLink,
+} from 'grommet-icons';
+
+const defaultFormValues = {
+  'text-input-validation': '',
+  'radio-button-group-validation': '',
+  select: '',
+  checkboxgroup: '',
+  'text-area': '',
+};
+
+const stepOneValidate = values => {
+  const emailRegex = RegExp(
+    '[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+',
+  );
+  const emailValid = emailRegex.test(values['text-input-validation']);
+
+  return {
+    email: emailValid ? '' : 'Invalid email address.',
+    isValid: !!emailValid,
+  };
+};
+
+const StepOne = ({ active, setActive, formValues }) => {
+  const [error, setError] = useState({
+    email: '',
+    radiobuttongroup: '',
+  });
+
+  return (
+    <>
+      <Box margin={{ bottom: 'large' }}>
+        <FormField
+          label="Email"
+          htmlFor="text-input-validation"
+          name="text-input-validation"
+          error={error.email}
+          onChange={() => setError(stepOneValidate(formValues))}
+        >
+          <TextInput
+            placeholder="jane.smith@hpe.com"
+            id="text-input-validation"
+            name="text-input-validation"
+            type="email"
+          />
+        </FormField>
+        <FormField
+          help="Click on the ‘eye’ icon for the description text field to hide."
+          htmlFor="radio-button-group-validation"
+          label="RadioButtonGroup"
+          name="radio-button-group-validation"
+        >
+          <RadioButtonGroup
+            id="radio-button-group-validation"
+            name="radio-button-group-validation"
+            options={['Radio button 1', 'Radio button 2']}
+          />
+        </FormField>
+      </Box>
+      <Button
```

Comment:
Feeling like we need some treatment on or near the next button letting the user know there is an error (Like the error message @britt6612 has been working on). Or maybe not, if we assuming the error will be always visible in a wizard context? Can we make that assumption? Probably not on mobile?

Thinking aloud here.

Reply:
That is a good point. I think adding something similar to what Brittany has implemented would be effective.

Reply:
Implemented.